### PR TITLE
[MODULAR] Blacklists morphine and all inverses from smartdarts.

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

For the darts that are supposed to have no combat use (doing no damage when hitting someone, and letting pacifists use them), letting them load two of the more-effective 'get fucked' chems, is a 'bit' much.

## How This Contributes To The Skyrat Roleplay Experience

No more mass-printable super-melatonin/ nooartrium darts.

## Changelog

:cl:
balance: Smartdarts can no longer load super-melatonin, or morphine, relegating them back to the actual combat-oriented syringe guns.
/:cl:

### CODE REVIEW

**Review by Zonespace27 — APPROVED** [review]
Might want to consider adding regular Tirimol to it but that's iffy

**Comment by RatFromTheJungle** [comment]
> Might want to consider adding regular Tirimol to it but that's iffy

maybe, but for this PR, i'd want to stick to just the 'just combat' stuff

**Comment by softcerv** [comment]
> Might want to consider adding regular Tirimol to it but that's iffy

Regular Tirimol is fine considering it takes twice the amount of cycles as Super Melatonin to knock someone out. SmartDarts should still be able to be used for self-defense, but people should be more creative about the way that they end up doing it.

File: modular_skyrat/modules/medical/code/smartdarts.dm [hunk 1]
```diff
@@ -83,6 +83,10 @@
 		/datum/reagent/medicine,
 		/datum/reagent/vaccine
 	)
+	var/list/disallowed_medicine = list(
```
Comment (softcerv) [inline modular_skyrat/modules/medical/code/smartdarts.dm:87]: Autodoc
```suggestion
        ///Blacklist that contains medicines that SmartDarts are unable to inject.
	var/list/disallowed_medicine = list(
```

File: modular_skyrat/modules/medical/code/smartdarts.dm [hunk 2]
```diff
@@ -83,6 +83,11 @@
 		/datum/reagent/medicine,
 		/datum/reagent/vaccine
 	)
+        ///Blacklist that contains medicines that SmartDarts are unable to inject.
+	var/list/disallowed_medicine = list(
+		/datum/reagent/inverse/healing/tirimol,
```
Comment (YakumoChen) [inline modular_skyrat/modules/medical/code/smartdarts.dm]: Does this even load in? It's already not part of reagent/medicine or reagent/vaccine

Comment (RatFromTheJungle) [reply modular_skyrat/modules/medical/code/smartdarts.dm]: it does

**Comment by OzzyMcPseudonym** [comment]
Lowkey might want to consider adding Hercuri. That shit slows you down in a very small amount of time, which is usually as much of a death sentence as a sleep to be honest with you. That and ever-increasing burn damage from extreme cold doesn't help either.

**Pull request merged**